networkx/utils/misc.py

```diff
@@ -194,7 +194,7 @@ def arbitrary_element(iterable):
     This is most useful for "peeking" at an arbitrary element of a set,
     but can be used for any list, dictionary, etc., as well::
 
-        >>> arbitrary_element({3, 2, 1})
+        >>> arbitrary_element([1, 2, 3])
```

Comment:
According to https://github.com/networkx/networkx/issues/2203#issuecomment-236320863 , the best way would be to ignore all doctests.